Makefile

```diff
@@ -216,34 +216,34 @@ clean:
 	mkdir -p $(TOPDIR)/bin
 
 obj/image-crom.bin:
-	${LD} -o obj/image-crom.elf ${OBJECTS-CROM} ${RESOURCES} ${LDFLAGS-ROM}
-	${OBJCOPY} --output-target=binary --strip-all obj/image-crom.elf $@
+	@${LD} -o obj/image-crom.elf ${OBJECTS-CROM} ${RESOURCES} ${LDFLAGS-ROM}
+	@${OBJCOPY} --output-target=binary --strip-all obj/image-crom.elf $@
```

Comment:
With these being removed, is there *any* status update which tells the user what is currently happening?

If errors / warnings arise, how would the user know what they are about, if we hide the command that generated them?

Reply:
Warnings and errors are still properly displayed, and as stated in the PR desc line outputs for what folder make is currently building in still show.